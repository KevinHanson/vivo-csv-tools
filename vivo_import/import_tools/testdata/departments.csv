URI,Department
1000,"Anatomy"
1001,"Anesthesiology"
1002,"Medicine"
