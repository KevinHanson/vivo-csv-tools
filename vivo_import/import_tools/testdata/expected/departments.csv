1002,Medicine,
1000,Anatomy,
1001,Anesthesiology,
